test/integration/features/coolstore.feature

```diff
@@ -33,7 +33,7 @@ Feature: Cool Store
      """
      Success
      """
-    When services "web-ui, inventory, catalog, cart, coolstore-gw" rollout successfully
+    When services "web-ui, inventory, catalog, cart, coolstore-gw" rollout successfully in "1200" seconds
```

Comment:
@agajdosi So now we are using around 20 mins for coolstore feature? Also do we have a dry run of this integration test on CI so we can have a look how it error out in case something fails or timeout?

Reply:
@praveenkumar Yes, now the test will fail after 20 minutes (we can tune it if needed) and continue, so it won't get stuck. We can setup a dry run of coolstore feature on nightlies and see what's going on with coolstore.